k8s/deploy-storageos/cluster-operator/manifests/040-operator-deployment.yaml

```diff
@@ -18,7 +18,7 @@ spec:
       serviceAccountName: storageoscluster-operator-sa
       containers:
         - name: storageoscluster-operator
-          image: "storageos/cluster-operator:v1.0.0"
+          image: "storageos/cluster-operator:1.1.0"
```

Comment:
Can you double check this, it seems that previously it was `v1.x.x` and now is `1.x.x` ? Is the `v` missing or has it been dropped permanently?